obs/packages.go

```diff
@@ -0,0 +1,156 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package obs
+
+import (
+	"bytes"
+	"context"
+	"encoding/xml"
+	"fmt"
+	"net/http"
+	"net/url"
+)
+
+// Package contains information about an OBS Package
+type Package struct {
+	XMLName     xml.Name `json:"directory" xml:"directory"`
+	Name        string   `json:"name" xml:"name,attr"`
+	Title       string   `json:"title,omitempty" xml:"title,omitempty"`
+	Description string   `json:"description,omitempty" xml:"description,omitempty"`
+	Devel       Devel    `json:"devel" xml:"devel"`
+	Entries     []Entry  `json:"entry" xml:"entry"`
```

Comment:
I don't see this present in API endpoints for creating/updating and getting packages, I recommend removing it for now or eventually marking it as `omitempty`

Reply:
It's part of the meta file for packages - https://api.opensuse.org/apidocs/#/Sources%20-%20Packages/get_source__project_name___package_name___meta